05_vectors_and_result_enum.md

```diff
@@ -87,22 +88,21 @@ Let's examine what `hex::decode` returns.
 Remember, we want to work with a Vector of bytes so a `vec<u8>` is the data type we're looking for.
 However, if we look at the return type of the `decode` function we see that the data structure we want is wrapped *inside* of a `Result` type.
 
-
 The `Result` type is a common `enum` that you will see in Rust code.
-Enums are a way to describe a mutually exclusive set of options for a particular variable.
-If you think about it, it's possible that the `hex::decode` function fails to return a proper collection of bytes.
+Enums are a way to describe a mutually exclusive set of values for a particular variable.
+If you think about it, the `hex::decode` function can fail to return a proper collection of bytes.
 For example, what if one of the characters is not a hex character?
 So we get two possibilities from a `Result`, an `Ok` response or an `Err` response.
+The former represents a successful computation whereas the latter represents that an error occurred.
```

Comment:
might read easier if you replace the "represents that" with "indicates"
"...whereas the latter indicates an error occurred"

Reply:
Agree